index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<!-- Link CSS -->
+	<link rel="stylesheet" href="./styles/style.css">
+	<!-- Font Link -->
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+	<title>Document</title>
+</head>
+
+<body>
+
+	<header>
+		<div class="nav-bar">
+			<a href="#">Hernán Zamora</a>
+			<div class="menu">
+				<div class="bar"></div>
+				<div class="bar"></div>
+				<div class="bar"></div>
+			</div>
+		</div>
+	</header>
+
+	<main>
+		<div class="main-container">
```

Comment:
- [ ] Kindly ensure that the container of the headline section is wrapped in a `section` tag and not a `div` tag because the section tag * represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.